src/app/account/settings/ExportTransactions/index.tsx

```diff
@@ -0,0 +1,74 @@
+"use client"
+import React, { use, useEffect, useState } from 'react'
+import { LocalStorage, decryptTransactions, getTransactionsFromSupabase, getUserId } from '@/lib/utils';
+import { useSupabase } from '@/app/supabase-provider';
+import { SupabaseClient } from '@supabase/auth-helpers-nextjs';
+import { useToast } from '@/app/components/ui/use-toast';
+
+function objectArrayToCsvString(array:any[],delimiter:string=',',headers:any[]){
```

Comment:
Can you please check why `prettier` is not working for you? It's not formatting the document properly.